lib/Cake/Test/Case/Model/Datasource/Database/MysqlTest.php

```diff
@@ -3701,6 +3704,7 @@ public function testFieldsWithComplexVirtualFields() {
  * @return void
  */
 	public function testExecute() {
+		$this->loadFixtures('Article');
```

Comment:
The value of `$autoFixtures` in MysqlTest is set to `false`. According to the [documentation](https://api.cakephp.org/2.9/class-CakeTestCase.html#$autoFixtures) the fixtures must be loaded manually here:
```
/**
  * By default, all fixtures attached to this class will be truncated and reloaded after each test.
  * Set this to false to handle manually
  *
  * @var array
  */
public $autoFixtures = true;
```
So the missing call to `loadFixtures` here is actually a bug. I believe that the code worked without loading fixtures manually because the fixtures were not properly dropped in `CakeFixtureManager::unload` as it is promissed in the [documentation](https://api.cakephp.org/2.9/class-CakeTestCase.html#$dropTables):
```
/**
  * Control table create/drops on each test method.
  *
  * Set this to false to avoid tables to be dropped if they already exist
  * between each test method. Tables will still be dropped at the
  * end of each test runner execution.
  *
  * @var bool
  */
public $dropTables = true;
```